docs/antora.yml

```diff
@@ -11,7 +11,7 @@ asciidoc:
     xrefstyle: short@
     listing-caption: Example@
     example-caption: false
-    iso-date-format-link: 'https://en.wikipedia.org/wiki/ISO_8601[ISO-8601]'
+    iso-date-format-link: 'https://en.wikipedia.org/wiki/ISO_8601#Durations[ISO-8601 Durations]'
```

Comment:
some of links may require updates with this change, for example:
https://github.com/vividus-framework/vividus/blame/master/docs/modules/plugins/pages/plugin-db.adoc#L321